src/runtime/manifest.ts

```diff
@@ -52,32 +53,32 @@ export interface ManifestEndpoint {
   callableTrigger?: {};
 
   eventTrigger?: {
-    eventFilters: Record<string, string>;
-    eventFilterPathPatterns?: Record<string, string>;
+    eventFilters: Record<string, string | Expression<string>>;
+    eventFilterPathPatterns?: Record<string, string | Expression<string>>;
     channel?: string;
     eventType: string;
-    retry: boolean;
+    retry: boolean | Expression<string>;
```

Comment:
Shouldn't this be `Expression<boolean>`?

Reply:
Yes. Fixed.